lib/models.js

```diff
@@ -1612,9 +1948,44 @@ class ElementMapping {
 
     return clone;
   }
+
+  toJSON() {
+    let fieldMappings = this.rulesFilter.field._rules.reduce((dict, m) => {      
+      var paths = m.sourcePath.map(p => p.fqn).slice();
```

Comment:
Is the `.slice()` necessary?  Doesn't `.map(...)` return a new array anyway?  I don't see why we'd need to make a copy.

Reply:
I don't even think this part of the code is being used. It was part of an incomplete effort to make the "fieldMapping" hierarchical. I'll comment this block out and leave a todo statements (unless you think it's more appropriate to remove it altogether for the release and stash it in a new branch.

Reply:
I'm fine w/ commenting it out for now if you think we'll use it later.